.babelrc

```diff
@@ -1,7 +1,19 @@
 {
-    "presets": ["es2015", "flow"],
+    "presets": [
+        ["es2015", {
+            "modules": false
+        }],
+        "flow"
+    ],
     "plugins": [
         "transform-runtime",
         "transform-class-properties"
-    ]
+    ],
+    "env": {
+        "test": {
+            "plugins": [
+                "transform-es2015-modules-commonjs"
```

Comment:
This is for Jest, as it [doesn't support ES6 modules](https://facebook.github.io/jest/docs/en/getting-started.html).
In Babel 6, env configurations doesn't overrides configuration, rather are merged to the root configuration(babel/babel#5276).

Reply:
Could you avoid adding this dependency by adding the `.default` to your imports instead?

Reply:
@rrandallcainc Jest doesn't understand ES6 `import`/`export` syntaxes, so I think adding `.default` won't help it. Also this plugin is part of `babel-preset-es2015`, so there won't be any additional dependencies.